py/autoevals/util.py

```diff
@@ -10,3 +10,41 @@ def as_dict(self):
     def as_json(self, **kwargs):
         """Serialize the object to JSON."""
         return json.dumps(self.as_dict(), **kwargs)
+
+
+class NoOpSpan:
```

Comment:
We can import this from logger too.

Reply:
but what if the user isn't using braintrust? we'll still need to return something from `current_span()`